### PR TITLE
Error tweaks

### PR DESCRIPTION
**Pull request by pydsigner**
Some deduplication and message tweaks

### CODE REVIEW

File: py3status/modules/thunderbird_calendar.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def _error_response(self, text, color):
         response = {
             'cached_until': time() + self.cache_timeout,
             'full_text': text,
-            'color': color
+            'color': 3s_config['color_bad']
```
Comment (mrt-prodz) [inline py3status/modules/thunderbird_calendar.py]: There is a small issue here where i3s_config is not declared yet, that's why I was passing the color as a parameter.

We could change the color parameter to i3s_config and pass that object to _error_response maybe? Would be less verbose than passing 'color_bad' each time.

`def _error_response(self, text, i3s_config):`

Comment (pydsigner) [inline py3status/modules/thunderbird_calendar.py]: You're right, let me rework this a little.

**Comment by pydsigner** [comment]
Should be better now.

File: py3status/modules/thunderbird_calendar.py [hunk 2]
```diff
@@ -60,22 +63,18 @@ def get_calendar(self, i3s_output_list, i3s_config):
...
-                'cached_until': time() + self.cache_timeout,
-                'full_text': self.format.format(due=due,
-                                                completed=completed,
-                                                current=current)
-            }
-            return response
+            
+            return self._response(self.format.format(due=due,
+                                                     completed=completed,
+                                                     current=current)
```
Comment (mrt-prodz) [inline py3status/modules/thunderbird_calendar.py]: Thanks a lot for the help this is much better, there is just a missing closing parenthesis on L75. (beside that everything is working fine here)

edit: It's actually L73 on commit 230d839

Comment (pydsigner) [inline py3status/modules/thunderbird_calendar.py]: Wow. I better not do much coding today, errors right and left....

**Comment by mrt-prodz** [comment]
Thanks a lot @pydsigner !

**Comment by mrt-prodz** [comment]
I'm checking quickly why flake8 is complaining.

**Pull request merged**